tests/outputs/expected/sliced_housing_multiple.py

```diff
@@ -13,6 +13,7 @@ def is_new(col):
     x = assets[["SalePrice", "Lot_Area", "Garage_Area"]]
     clf.fit(x, y)
     p = clf.predict([[100 * 1000, 10, 4]])
+    pickle.dump(p, open("/tmp/fake", "wb"))
```

Comment:
don't see the `import pickle`

Reply:
good catch, will add a subprocess run to test the script_dag so that we catch these in the future. the script_dag should be runnable.

Reply:
added a test to run the script to see if the script_dag we generate actually works. its a longer test because its installing pandas and sklearn to run it but we could probably do this either without the virtual env or make a simpler test that does not depend on pandas and sklearn